src/RF24Logger.h

```diff
@@ -22,6 +22,13 @@
 #include "RF24LogLevel.h"
 #include "RF24LogHandler.h"
 
+#if defined (ARDUINO_ARCH_AVR)
+  #define RF24LOG_FLASHIFY(A) F(A)
+  #define RF24LOGGER_info(vendorId, message, ...) (rf24Logger.log(RF24LogLevel::INFO, RF24LOG_FLASHIFY(vendorId), RF24LOG_FLASHIFY(message), ##__VA_ARGS__))

```

Comment:
On AVR this macro will wrap the provided texts in ```F()``` macro. This will save RAM and store the texts in FLASH. usage of this macro doesn't depend on architecture. The macro will be resolved during compilation time.